src/main/resources/update/3_13.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE WEB_HOOK ADD COLUMN TOKEN VARCHAR(100);
+ALTER TABLE WEB_HOOK_EVENT ADD COLUMN TOKEN VARCHAR(100);
```

Comment:
It should not ; I even do not remember why I needed that ; it makes no sense neither to me now.
I'll review the PR tonight.
Thanks for opening my eyes:  less is better... often!